packages/terra-consumer-layout/src/Logo.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```

Comment:
can we move this to the tests folder? This isn't a logo we need to ship with, it should be provided by the consumer

Reply:
weird, I remember I deleted. should be deleted now.